src/lib/init.js

```diff
@@ -38,9 +38,19 @@ function readExternalConsentData(config) {
 
 function readInternalConsentData() {
 	return Promise.all([
-		readVendorConsentCookie(),
-		fetchPubVendorList(),
-		readPublisherConsentCookie()
+		[
+			readVendorConsentCookie(),
+			fetchPubVendorList(),
+			readPublisherConsentCookie()
+		],
+		undefined
```

Comment:
`undefined` here is unnecessary, because if we didn't provide argument `globalVendorConsentData` it will be `undefined` by default.